packages/transactions/tests/keys.test.ts

```diff
@@ -70,10 +70,10 @@ test('Retrieve public key from signature', () => {
   const compressedPubKey = '03ef788b3830c00abe8f64f62dc32fc863bc0b2cafeb073b6c8e1c7657d9c2c3ab';
 
   const message = 'hello world';
-  const sig = signWithKey(privKey, message);
```

Comment:
Did `signWithKey` used to work with non-hex encoded strings? If so, is this a change in that method's semantics?